netmiko/ssh_autodetect.py

```diff
@@ -151,15 +163,9 @@
         "priority": 99,
         "dispatch": "_autodetect_std",
     },
-    "brocade_netiron": {
-        "cmd": "show version",
-        "search_patterns": [r"NetIron"],
-        "priority": 99,
-        "dispatch": "_autodetect_std",
-    },
-    "extreme_slx": {
+    "ruckus_fastiron": {
         "cmd": "show version",
-        "search_patterns": [r"SLX-OS Operating System Software"],
+        "search_patterns": [r"Ruckus", r"Foundry Networks"],
```

Comment:
That seems too general. 

Those are two independent searches i.e. the above would match on any "show version" that contained "Ruckus" and any "show version" that contained "Foundry Networks"